app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_likes_counter
+  after_destroy :update_likes_counter
+
```

Comment:
Great job @RaphDasilva 🤝

- [ ] [Optional] You may also make the `update_likes_counter` method private.